install/traefik-install.sh

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 tteck
+# Author: tteck (tteckster)
+# License: MIT
+# https://github.com/tteck/Proxmox/raw/main/LICENSE
+
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y curl
+$STD apt-get install -y sudo
+$STD apt-get install -y mc
+$STD apt-get install -y gpg
+$STD apt-get install -y apt-transport-https
+msg_ok "Installed Dependencies"
+
+RELEASE=$(curl -s https://api.github.com/repos/traefik/traefik/releases/latest | grep "tag_name" | awk '{print substr($2, 3, length($2)-4) }')
+msg_info "Installing Traefik v${RELEASE}"
+mkdir -p /etc/traefik/{conf.d,ssl}
+wget -q https://github.com/traefik/traefik/releases/download/v${RELEASE}/traefik_v${RELEASE}_linux_amd64.tar.gz
+tar -C /tmp -xzf traefik*.tar.gz
+mv /tmp/traefik*/traefik /usr/bin/
+rm -rf traefik*.tar.gz
+msg_ok "Installed traefik"
+
+msg_info "Creating Service"
+cat <<EOF >/etc/systemd/system/Traefik.service
+[Unit]
+Description=Traefik is an open-source Edge Router that makes publishing your services a fun and easy experience
+
+[Service]
+Type=notify
+ExecStart=/usr/bin/traefik --configFile=/etc/traefik/traefik.yaml
+Restart=on-failure
+ExecReload=/bin/kill -USR1 $MAINPID
+
+[Install]
+WantedBy=multi-user.target
+EOF
+
+systemctl enable -q --now Traefik.service
+msg_ok "Created Service"
+
+msg_info "Creating traefik configuration"
```

Comment:
Should "Creating Traefik configuration" come before "Creating Service"? At this point traefik has already started.
Capital "T"